tests/e2e/README.md

```diff
@@ -24,8 +24,8 @@ DEBUG=true
 For local setup:
 
 1. Create file `local.env` in the `tests/e2e/config` folder with required values.
-    * If you have access to the Subscriptions plugin, please follow the instructions from `tests/e2e/specs/subscriptions/README.md` before running the setup. Make sure to include also Action Scheduler plugin from the same document.
-    * If you don't, you may skip Subscriptions setup and tests by adding `SKIP_WC_SUBSCRIPTIONS_TESTS=1` to your `local.env` in the `tests/e2e/config` folder.
+    * If you have access to the Subscriptions plugin, please follow the instructions from `tests/e2e/specs/README.md` before running the setup. Make sure to include also Action Scheduler & WooCommerce Blocks plugins from the same document aswell.
```

Comment:
```suggestion
    * If you have access to the Subscriptions plugin, please follow the instructions from `tests/e2e/specs/README.md` before running the setup. Make sure to include also Action Scheduler & WooCommerce Blocks plugins from the same document as well.
```

Reply:
nitpick :)

Reply:
What it means `nitpick`? :) @oaratovskyi

Reply:
Ah, it means that `a change is small and may not be very important, but is technically correct.` 😅

I've found a couple of good links about that
https://stackoverflow.com/a/27810540
https://en.wikipedia.org/wiki/Nitpicking

Reply:
Aha, good. Thanks for sharing these links :)

So, looks like this is ready to be merged. I'll merge after the checks. Did changes that you've mentioned.

Reply:
thanks, @veljkho for addressing those 👍